### DISCUSSION TITLE
Is the permissions chart meant to be the interactive?

### DISCUSSION THREAD

**Discussion by Immortalin** [supabase/supabase#249] [Questions] [answered]
Is the permissions chart meant to be the interactive? Is it like Hasura?

**Comment by kiwicopple** [comment, marked as answer]
Hey @Immortalin - at the moment it's just indicative. We will add functionality later for this, but you don't really need it since most of the time you will just be able to use the "Auth" section in Supabase, which works with Row Level Security 

> Is it like Hasura?

Postgres roles/user grants are a bit different from Hasura. In [Postgres](https://www.postgresql.org/docs/current/user-manag.html), roles & users are essentially the same thing except that "users" can log in to the Database. Roles can have permissions, and can inherit from other roles.

For now, I would suggest you ignore this section and instead focus on the "Policies" tab of the Authentication section. We will work on Postgres roles/users/permissions in the future.

**Reply by edwardLz** [reply]
I can somehow create a schema in a stored procedure, from client?

**Reply by silentworks** [reply]
@edwardLz as what you are asking or saying is not related to the original discussion, can you create a new discussion topic please? also can you make it clear if you are asking if there is a way to create a schema from a stored procedure or are you saying that this is possible and it shouldn't be?

**Reply by edwardLz** [reply]
ok! there is #957
